tests/Server/Test_ScillaIPCServer.cpp

```diff
@@ -68,10 +69,10 @@ BOOST_AUTO_TEST_CASE(test_connection) {
 
 // Simple non-map query.
 BOOST_AUTO_TEST_CASE(test_query_simple) {
-  UnixDomainSocketServer s(SCILLA_IPC_SOCKET_PATH);
+  rpc::UnixDomainSocketServer s(SCILLA_IPC_SOCKET_PATH);
```

Comment:
Perhaps move this to some fixtures class so it doesn't have to be repeated for each test?